### PR TITLE
Add some basic checks which verify that a no default private and public key is installed for stanley

### PR DESCRIPTION
**Pull request by Kami**
This pull request adds some basic checks which verify that a no default private key and public key is installed for stanley user after install.sh script completes.

As an additional check / improvement we should also add some tests which verify that the key is indeed installed when `st2::stanley::ssh_private_key` value is set in the answers file.

### CODE REVIEW

**Comment by estee-tew** [comment]
https://stackstorm.slack.com/archives/stackstorm/p1452276280021714

File: actions/workflows/st2workroom_test.yaml [hunk 1]
```diff
@@ -76,7 +76,29 @@
         distro: "{{distro}}"
         user: "{{st2_username}}"
         pass: "{{st2_password}}"
+      on-success: "verify_no_default_private_ssh_key_is_installed_for_stanley_1"
```
Comment (manasdk) [inline actions/workflows/st2workroom_test.yaml]: Could these test be a separate workflow simply called from here? I suspect these will grow.

File: actions/workflows/st2workroom_test.yaml [hunk 2]
```diff
@@ -76,7 +76,29 @@
...
+      params:
+        hosts: "{{hostname}}"
+        cmd: 'files=$(find /home/stanley/.ssh/ -type f ! -name authorized_keys | wc -l) ; if [ $files -gt 0 ]; then echo "Found SSH key files"; exit 1; fi'
+      on-success: "verify_no_default_authorized_key_is_installed_for_instaley"
+    -
+      name: "verify_no_default_authorized_key_is_installed_for_instaley"
+      ref: "core.remote_sudo"
+      params:
+        hosts: "{{hostname}}"
+        cmd: 'lines=$(cat /home/stanley/.ssh/authorized_keys | grep st2_stanley_key | wc -l) ; if [ $lines -gt 0 ]; then echo "Found default stanley in authorized_keys"; exit 1; fi'
```
Comment (manasdk) [inline actions/workflows/st2workroom_test.yaml]: lol ... this constitutes an abuse in my view :P

Comment (Kami) [inline actions/workflows/st2workroom_test.yaml]: I agree, I'm not a fan of it, but right now there is really no nicer way to do it.

I was thinking about putting those checks in script files and rsyncing / scping those files over and directly calling them.

Comment (Kami) [inline actions/workflows/st2workroom_test.yaml]: Actually, I will try porting it remote runner script action and see how it goes.

I wanted to do that in the past with st2enterprise tests, but I had some issues.

**Comment by Kami** [comment]
I'll go ahead, merge this into master and work from there.

The whole develop / test loop is already slow and having this change in a branch makes it even slower.

File: actions/workflows/st2workroom_test.yaml (hunk 2 above)
Comment (Kami) [inline actions/workflows/st2workroom_test.yaml]: Much better now - https://github.com/StackStorm/st2cd/pull/102/files#diff-b42acea6d62fe3c15c64402a82b63f13

**Pull request merged**